docs/mevblocker/concepts/Order Flow Auction/order-flow-auction.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 3
+---
+# MEV Blocker OFA
+
+MEV Blocker's Transaction Flow can be broken down into 6 major steps. In the picture below you can find a general overview of the flow of a transaction.
+
+![The Best Protection Under the Sun](/img/mevblocker/mevblocker_ofa.png)
+
+## User submits the transactions
+
+This is the first step, where the user creates the transaction:
+
+1. If you are a normal user in which you have 100% control of all settings, you must first have installed the custom MEV Blocker RPC in your wallet. Learn more about how to do it here
+2. If you are a wallet that sends the transaction on behalf of the user, then your code must have integrated the custom MEV Blocker RPC. Learn more about how to do it here
+3. If you are a dApp that sends the transaction on behalf of the user - for example, because you use intents like CoW Swap - then your code must have integrated the custom MEV Blocker RPC. Learn more about how to do it here
+
+* Otherwise, you can prompt your users to protect themselves by adding MEVBlocker using this simple javascript
```

Comment:
```suggestion
* Otherwise, you can prompt your users to protect themselves by adding MEV Blocker using this simple JavaScript
```

Also, where is the link to the JavaScript?

Reply:
Need to delete